bigbluebutton/bigbluebutton
# fix: allowModsToUnmuteUsers param not working

### What does this PR do?

restores missing `muted` field in userlist query, used by unmute user feature.

### Closes Issue(s)
Closes #20879

### How to test
1. Create a meeting with `allowModsToUnmuteUsers=true` param;
2. Join a mod and an attendee;
3. Join audio with attendee;
4. Attendee mutes itself;
5. Mod click on the user item in the user list;
6. Should display "Unmute user" button;
